spring-auto-restdocs-docs/section-snippet.adoc

```diff
@@ -109,4 +113,13 @@ AutoDocumentation.sectionBuilder()
     .build()
 ----
 <1> only selected snippets will be used in the specified order
-<2> if snippets having no content should be skipped from section
+<2> whether snippets having no content should be skipped from section
+
+==== Custom snippet in section
```

Comment:
it's actually custom snippet inside a section snippet. it's confusing because section snippet contains multiple snippet "sections" :-/